### PR TITLE
Add Chinese translation

### PR DESCRIPTION
**Pull request by rongpenl**
Addressing issue #368 . Probably need a second pair of eyes of a Chinese speaker to review but I am pretty confident about its accuracy and readability.

### CODE REVIEW

**Comment by rongpenl** [comment]
@GreatYYX wonder you have some time for a review?

**Comment by GreatYYX** [comment]
Nice job @rongpenl, the translation is precise and neat.

One suggestion from my point of view is that the APIs and terminologies could be kept in English (or put them aside): e.g, `用推特的 状态/过滤 API 来搜集实时推文` to be `用推特的 statuses/filter  (状态/过滤) API 来搜集实时推文`.

File: README_zw_zh.md [hunk 1]
```diff
@@ -0,0 +1,548 @@
...
+
+尤其需要注意的是 `search` 返回的是过去七天内的推文：这是推特搜索 API 的限制。如果你觉得这太短了——我们也觉得——你或许会更愿意尝试使用下文提到的 `filter` 和 `sample` 命令。
+
+最好的快速上手推特搜索语法的方法是实验[推特高级搜索](https://twitter.com/search-advanced)这个页面上的样例。你可以复制粘贴搜索框里的查询语句。比如这里有一个比较复杂的查询语句，它搜索包含有 `#blacklivesmatter` 和 `#blm` 关键字并发给 [deray](https://twitter.com/deray) 的推文。
+
+```shell
+twarc search '#blacklivesmatter OR #blm to:deray' > tweets.jsonl
+```
+
+你还应当看一看 Igor Brigadir 关于推特高级搜索语法`精彩绝伦`的指南: [推特高级搜索 (英文)](https://github.com/igorbrigadir/twitter-advanced-search/blob/master/README.md). 这份指南里包含了很多阅读推特搜索文档后依然不显然的玄妙之处。
```
Comment (GreatYYX) [inline README_zw_zh.md:81]: The style for word "excellent" is Italic not code (escaping backticks).

**Comment by edsu** [comment]
This is very impressive, thank you for your efforts @rongpenl, and for the review @GreatYYX! I took a look in [Google Translate](http://translate.google.com/translate?hl=en&sl=zh-CN&tl=en&u=https%3A%2F%2Fgithub.com%2Frongpenl%2Ftwarc%2Fblob%2Fmain%2FREADME_zw_zh.md&sandbox=1) to get the feel for the changes, and was really impressed with the coverage(even though I'm sure the translation is imperfect).

I will go ahead and merge this now, but please feel free to submit additional PRs with updates!

**Pull request merged**